libs/ui/src/components/NeoInput/NeoInput.scss

```diff
@@ -26,13 +25,13 @@ $input-textarea-padding: 0.5rem;
     }
   }
 
-    .o-field__label {
-      margin-bottom: 0.5rem;
-      font-weight: 400;
-      @include ktheme() {
-        color: theme('text-color');
-      }
+  .o-field__label {
+    margin-bottom: 0.5rem;
+    font-weight: 400;
```

Comment:
can be replaced with:

$field-label-font-weight: 400;
$field-margin-bottom: 500;